docs/nodes/qgb.md

```diff
@@ -0,0 +1,203 @@
+transactions.
+
+`keyring-account` refers to the validator key name on Celestia.
+
+Running the orchestrator command above will show logs like the following
+output:
+
+```sh
+SHOW LOGS
+```
+
+With that, you have started the orchestrator! Now, let's move on to the Relayer.
+
+PS: Validators don't need to run relayers or deploy the QGB smart contract.
+They can, but it is not a must.
+
+### Using the Relayer to Deploy the QGB Smart Contract
+
+Open a new terminal session in order to deploy the QGB Smart Contract.
+
+> NOTE: This step is necessary if no QGB Smart Contract is deployed on an EVM
+  Chain. Here, we are deploying to the Ethereum Sepolia Testnet.
+
+<!-- markdownlint-disable MD013 -->
+```sh
+celestia-appd deploy -x "test" -d <EVM-PRIVATE-KEY> -e https://rpc.sepolia.org -z 11155111
+```
+<!-- markdownlint-enable MD013 -->
+
+For this to work, you will need Sepolia tokens on your EVM address you generated
+earlier. You can use the Sepolia Faucet to get Sepolia tokens [here](https://sepolia-faucet.pk910.de/).
+
+The `-e` flag references the RPC endpoint for Sepolia in order to connect to it.
+
+The `-z` flag references the Chain ID of Sepolia.
+
+When you run the command, you should get the following output that indicates
+the QGB Smart Contract has been deployed:
+
+```sh
+SHOW LOGS
+```
+
+The logs will show you both the smart contract address for the deployed QGB
+smart contract on Sepolia as well as the transaction hash. You can use the
+Sepolia explorer to view the contract address [here](https://sepolia.etherscan.io/).
+
+With that, you have successfully deployed the QGB Smart Contract. Let's now
+run the Relayer.
+
+### Run the Relayer
+
+You can run the relayer in a separate terminal session with the following command:
+
+<!-- markdownlint-disable MD013 -->
+```sh
+celestia-appd relayer -a <QGB-CONTRACT-ADDRESS> -d <EVM-PRIVATE-KEY> -e https://rpc.sepolia.org -z 11155111  -l 25000000
+```
+<!-- markdownlint-enable MD013 -->
+
+The `-a` flag must be the QGB smart contract address that you received in the output
+when you deployed the smart contract in the previous section.
+
+The `-l` flag refers to the gas limit being set for the Relayer.
+
+Running the command will give you the following output:
+
+```sh
+SHOW LOG
+```
+
+With that, the QGB for your Validator is running. You can check the smart contract
+address on Sepolia Block Explorer to view the attestations being submitted to the
+smart contract.
+
+## Automated Network Generation
+
+You have a few resources you can use for instantiating a local QGB Network.
+
+### Shell Script
+
+You have the following script [here](https://github.com/celestiaorg/celestia-app/blob/main/scripts/single-node.sh)
+You can run it with the following command:
+
+```sh
+cd celestia-app/
+bash script/single-node.sh
+```
+
+This starts a local network with QGB enabled. You can then run the Orchestrator
+and Relayer after which were covered in the previous sections.
+
```

Comment:
Hmm.. should we replace the private key with <EVM_PRIVATE_KEY> @SweeXordious?

Reply:
That makes sense. But would require updating the single node script, since it has the corresponding address hardcoded. 
So, I guess for now, we can leave it that way.
But I'm fine if you decide to update the single node script, or copy the commands there to this doc and ask the devs to run them one by one.
Whichever you prefer :D

Reply:
Ahh - okay that makes sense. I'll ask you a little more about this next week 😄 thanks Rachid

Reply:
i added these suggestions. @YazzyYaz let's figure out the best way to deal with the private key on here though